canvas_modules/common-canvas/src/common-canvas/canvas-controller.js

```diff
@@ -1960,8 +1968,12 @@ export default class CanvasController {
 				break;
 			}
 			case "createAutoComment": {
-				const svgPos = this.commonCanvas.getSvgViewportOffset();
-				command = new CreateCommentAction(data, this.objectModel, svgPos);
+				const defaultCommentPosition = this.getDefaultCommentPosition();
+				data.mousePos = {
+					x: defaultCommentPosition.x_pos,
+					y: defaultCommentPosition.y_pos
+				};
+				command = new CreateCommentAction(data, this.objectModel);
```

Comment:
Could you change this so the defaultCommentPosition is passed into CreateCommentAction constructor as a parameter the set data.mousePos in the constructor please? It's just that I like to keep the case statements short and consistent with each other as much as possible.